exporters/jaeger/src/test/java/io/opentelemetry/exporters/jaeger/JaegerIntegrationTest.java

```diff
@@ -53,13 +55,22 @@
 
   @SuppressWarnings("rawtypes")
   @ClassRule
-  public static GenericContainer jaeger =
-      new GenericContainer("jaegertracing/all-in-one:" + JAEGER_VERSION)
-          .withExposedPorts(COLLECTOR_PORT, QUERY_PORT)
-          .waitingFor(new HttpWaitStrategy().forPath("/"));
+  @Nullable
+  public static GenericContainer jaeger = null;
+
+  static {
+    // make sure that the user has enabled the docker-based tests
+    if (Boolean.parseBoolean(System.getProperty("enable.docker.tests"))) {
```

Comment:
could be shortened to
```suggestion
    if (Boolean.getBoolean("enable.docker.tests"))) {
```